portafly/doc/ux_specs/product/overview.md

```diff
@@ -0,0 +1,17 @@
+# Product overview
+
+[Back to index](../../index.md)
+
+* [Mockups](https://marvelapp.com/prototype/ee82j74/screen/70939434)
+* [Mockups JIRA ticket](https://issues.redhat.com/browse/THREESCALE-4389)
+* [Parent JIRA ticket](https://issues.redhat.com/browse/THREESCALE-4197)
+
+##### Page header
+* Edit button
+  * Secondary button
+* Delete button
+  * Customized tertiary button
+
+##### Delete modal ([Global](../../global_components/modal.md))
+* [PatternFly Modal dialog design guidelines](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/modal)
```

Comment:
@deernano this should cover the specs for the delete modal (feel free to edit):
```
#### Delete modal ([specs](../../global_components/modal.md))
* Modal's title will include the name of the current Product the user is trying to delete
* Enter product name > text input field [required]
  * Validation error:
    * Wrong product name on _onblur_
* Buttons
  * Delete button [Disabled state until all conditions required to submit the form are met]
    * Primary danger button variant
    * Submits form
  * Cancel button [Always enabled]
    * Dismisses modal
```

@josemigallas please review.

**Note:** we should probably move some of these specs to `modal.md` so they can be reused for other contexts (backend deletion, user deletion etc), and only add the specs specific to this modal in this file

Reply:
Updated to Marvel: https://marvelapp.com/prototype/ee82j74/screen/71008634